core/common/src/main/java/alluxio/heartbeat/SleepingTimer.java

```diff
@@ -75,7 +76,7 @@ public void tick() throws InterruptedException {
         mLogger.warn("{} last execution took {} ms. Longer than the interval {}", mThreadName,
             executionTimeMs, mIntervalMs);
       } else {
-        mSleeper.sleep(mIntervalMs - executionTimeMs);
+        mSleeper.sleep(Duration.ofMillis(mIntervalMs - executionTimeMs));
```

Comment:
Are you thinking to move toward using `Duration` more in the codebase? Otherwise I think it is better to have the `sleeper` take milliseconds so we don't need to do this conversion in multiple places.